CMakeLists.txt

```diff
@@ -117,6 +118,49 @@ else ()
   message( FATAL_ERROR "OpenSSL was not found. Please install OpenSSL" )
 endif (OPENSSL_FOUND)
 
+if(NOT USE_SYSTEM_CURL)
```

Comment:
This should be in http-curl. If curl is disabled there is no point in adding it as an external project